futures.js

```diff
@@ -12,7 +12,7 @@ const S             = require('./common/sanctuary');
 
 //    readFile :: String -> Future Error String
 const readFile =
-S.compose(Future.node, S.curry3(fs.readFile, S.__, {encoding: 'utf8'}));
+S.curry3(Future.encaseN2, fs.readFile, S.__, {encoding: 'utf8'});
```

Comment:
I prefer the more concise `S.flip(Future.encaseN2(fs.readFile), {encoding: 'utf8'})`, why did you choose this option?